### DISCUSSION TITLE
Help need to create chart

### DISCUSSION THREAD

**Discussion by ruham20** [sqlitebrowser/sqlitebrowser#3148] [General]
Can anyone help me to create chart? I'm a beginner.
I need to do something like in image for my database.
Any help will be appreciated.
![image (1) (1)](https://user-images.githubusercontent.com/109856376/185698079-dd09c858-ac89-478b-b6da-456b65a56ce4.png)

**Comment by chrisjlocke** [comment]
Short answer ... not in DB4S.  While it has plot capabilities, it certainly can';t act on the data - popping up windows, etc.  You'll need to do extensive programming using the database.
It can be done, but it'd be a lot of work.
